src/scicookie/{{cookiecutter.project_slug}}/Makefile

```diff
@@ -60,11 +60,17 @@ clean: ## remove build artifacts, compiled files, and cache
 lint:
 	pre-commit run --all-files
 
-{% if cookiecutter.use_pytest %}
+{% if cookiecutter.use_pytest == "yes" -%}
 .PHONY:test
 test: ## run tests quickly with the default Python
 	pytest
-{% endif %}
+{%- endif +%}
+{% if cookiecutter.use_hypothesis == "yes" -%}
```

Comment:
one problem here is that, 
if pytest == yes and hypothesis == yes, it will create the test target twice.